cypress/support/pageObject/landingPage.js

```diff
@@ -1,12 +1,46 @@
 export class LandingPage {
-  interceptCall(NAME, METHOD, URL, STATUSCODE) {
-    cy.intercept({
-      method: METHOD, // "GET"
-      url: URL, // "https://api.realworld.io/api/tags"
-    }).as(NAME); // "Popular Tags"
-    cy.wait(`@${NAME}`).its("response.statusCode").should("eq", STATUSCODE);
+
+  interceptAndMockCallTags(name, method, url, statusCode, obj) {
+    cy.intercept(
+      {
+        method: method,
+        url: url,
+      },
+      (req) => {
+        req.reply({
+          statusCode: statusCode,
+          body: obj,
+        });
+      }
+    ).as(name);
+
+    cy.wait(`@${name}`).then(({ response }) => {
+      expect(response.statusCode).to.eq(statusCode);
+      expect(response.body).to.have.property(`${name}`);
```

Comment:
why do we need this? I mean if we are mocking data, why do we need to check that again? So as we spoke - we are mocking data to verify on FE what was shown, not the response it self